src/app.types.tsx

```diff
@@ -73,20 +68,23 @@ export interface QueryParams {
   sort: SortType;
   filters?: FiltersType;
   page: number;
-  startDate?: Date;
-  endDate?: Date;
+  dateRange: DateRange;
 }
 
 export interface RecordRow {
-  id: string;
-  shotNum: number;
   timestamp: string;
+  activeArea: string;
+  shotNum?: number;
+  activeExperiment?: string;
 
   [channel: string]: any;
 }
 
+export interface DateRange {
+  fromDate?: string;
+  toDate?: string;
+}
 // Update this whenever we have a new icon for a specific column
 export const columnIconMappings = new Map()
```

Comment:
Can we add some icons for `activeArea` and `activeExperiment`? For `activeExperiment` - `<Science />` seems like a good fit (a conical flask), as for `activeArea` - maybe `Location` (essentially the google maps pin)?

Reply:
Yeah can do! Since we know these columns are more likely to appear on screen, makes sense